MIP113/MIP113.md

```diff
@@ -772,9 +525,9 @@ The Governance Facilitators can propose expedited onboarding of Reserve Facilita
 
 The Governance Facilitators must put in place processes to monitor the Ecosystem Actors for risks of conflict of interest between Advisory Council Members and Active Ecosystem Actors, and ensure that the Scope Artifacts and the checks they place on Active Ecosystem Actors are in Universal Alignment with the Atlas.
 
-## 9: Interaction of Aligned Voter Committees (AVCs), Aligned Delegates (ADs), FacilitatorDAOs and Advisory Council
+## 9: Interaction of Aligned Delegates (ADs), FacilitatorDAOs and Advisory Council
 
-The Governance Facilitators must ensure to follow and enforce the principles described in *ATL2.9* to prevent misalignment of the core governance decision process.
+The Governance Facilitators must ensure to follow and enforce the principles described in the Atlas to prevent misalignment of the core governance decision process.
 
 ## 10: Core Governance Security
```

Comment:
10.1.1A below states that the GSM Pause Delay is 30h. Rune indicates 48 in the proposal.

Reply:
Where are these 48 hours, in the proposal posted in the forum? I see that the PR is using 30 hs for the GSM delay

Reply:
Yep, I fixed it in the PR